### ISSUE TITLE
Do binutils configure args --disable-doc and --disable-plugins do anything?

### ISSUE THREAD

**Issue by ryandesign** [autc04/Retro68#131]
build-toolchain.bash uses `--disable-doc` when configuring binutils for 68k or powerpc. Does this actually do anything? I cannot find this arg described in `binutils/configure --help`. I tried building binutils with and without this flag and the result seemed to be the same.

It also uses `--disable-plugins` when configuring binutils for powerpc. I also can't find this option described in `binutils/configure --help`. Does it actually do something?

**Comment by autc04** [comment]
At least they used to do something, but I might have upgraded GCC a few times since then.

`--disable-doc` was intended to disable things like building `info` documentation, which just adds to the dependencies and doesn't provide much value (I'm sure a lot more people these days know how to Google than how to use GNU info).

`--disable-plugins` was used to work around a problem caused by GCC's link-time optimization infrastructure. It might even work for 68K, which uses the well-supported ELF binary format, but the PowerPC target works via XCOFF. Binutils' XCOFF support is much less mature, and the GCC plugin caused some sort of trouble (I don't remember whether it was a build-time problem or simply a crash when the compiler was used...).

**Comment by ryandesign** [comment]
I agree with disabling documentation, but I can't yet find any evidence that `--disable-doc` does that for binutils.

I think `--disable-plugins` probably does still do what you want. I didn't see it described in the top-level binutils/configure script but I do see `--enable-plugins` described in `binutils/binutils/configure --help`. Projects like binutils that have multiple configure scripts are confusing. 😣

**Comment by autc04** [comment]
It's entirely possible that the flag did something some versions ago, and doesn't now. It is also entirely possible that I hallucinated the whole thing in the first place. I basically stopped paying attention to those flags as soon as the build script worked the first time. Also, I had a slower computer back then, so "let's do a complete rebuild to see if this flag is really necessary" wasn't my idea of fun ;-)
